pkg/tarmak/interfaces/interfaces.go

```diff
@@ -65,8 +65,7 @@ type Cluster interface {
 	// return public api hostname
 	PublicAPIHostname() string
 
-	// get status of Aamazon EBS encryption
-	AmazonEBSEncrypted() bool
+	Encrypted() bool
```

Comment:
Add a comment what that flags says. I don't fully understand what it means provider independent.